app/src/main/java/org/p2p/wallet/swap/ui/jupiter/main/JupiterSwapFragment.kt

```diff
@@ -124,13 +127,13 @@ class JupiterSwapFragment :
     }
 
     private fun setupWidgetsActionCallbacks() = with(binding) {
-        if (openedFrom == SwapOpenedFrom.OTHER) {
+        if (openedFrom == SwapOpenedFrom.BOTTOM_NAVIGATION) {
```

Comment:
Show keyboard if user entered from token details, top button actions view.
if swap is opened by bottom navigation, don't show the keyboard

Reply:
roger that